tests/Filesystem.php

```diff
@@ -25,8 +25,20 @@ public static function readFile(string $fileName): string
     /**
      * @return list<string>
      */
-    public static function find(string $pattern): array
+    public static function findRecursively(string $path, string $pattern): array
     {
-        return glob($pattern) ?: [];
+        /**
+         * @var iterable<string, \SplFileInfo> $fileIterator
+         */
+        $fileIterator = new \RecursiveIteratorIterator(new \RecursiveDirectoryIterator($path), \RecursiveIteratorIterator::CHILD_FIRST);
```

Comment:
I think these should be imported rather than `\FQCN`?

Reply:
A bit annoying we haven't set that up in php-cs-fixer... I for one keep remembering which one it should be.